cli/cmd/version.go

```diff
@@ -16,6 +16,10 @@ func Version() *cobra.Command {
 		Use:   "version",
 		Short: "Print the current version and exit",
 		Long:  `Print the current version and exit`,
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			version.InitBuild()
```

Comment:
could we instead move NewUsrbinSDK into this function and leave init() the way it was so the version package is as expected?